src/config.coffee

```diff
@@ -36,6 +36,40 @@ nconf.defaults
     params: []
   dataDir: SBS_HOME
 
+# Post-process custom files and params
+customFields = nconf.get('customFields')
+
+# Ensure array
+customFields.files = customFields.files || []
+# Always add upload_file_minidump file as first file
+customFields.files.splice(0, 0,
+  name: 'upload_file_minidump'
+  downloadAs: 'upload_file_minidump.#{id}.dmp'
```

Comment:
You could create a context with all the non-file parameters and template the string with that. I wouldn't expect someone to use anymore of handlebars than variable expansion, but it would let us expand any variable and maybe even supply helpers if the need arises.

If you don't want to do all this now, that's fine. Just change the template syntax to expand `{{id}}` so we have the option of adding that functionality later.